src/api/v1/jwt.go

```diff
@@ -0,0 +1,38 @@
+package v1
+
+import (
+	"crypto/rsa"
+	"fmt"
+	"github.com/ImpactDevelopment/ImpactServer/src/util"
+	"github.com/gbrlsnchs/jwt/v3"
+	"os"
+)
+
+var rs512 jwt.Algorithm
+
+func init() {
+	var key *rsa.PrivateKey
+
+	if env := os.Getenv("JWT_KEY"); env != "" {
+		var err error
+		key, err = util.StrToRsa(env)
+		if err != nil {
+			fmt.Println("WARNING: Unable to load JWT_KEY from the environment", err)
+		}
+	}
+
+	if key == nil {
+		fmt.Println("WARNING: JWT_KEY not specified, generating a temporary one")
+		key = util.GenerateRsa()
+	}
+
+	// rs512 can be used to sign and verify tokens, e.g. jtw.Sign(payload []byte, rs512 Algorithm)
+	rs512 = jwt.NewRS512(jwt.RSAPrivateKey(key), jwt.RSAPublicKey(&key.PublicKey))
```

Comment:
IMO: print out the public key in the same base64 format that gradle.properties is expecting, would be good for local running

Reply:
Good idea